contracts/tests/LineFactory.t.sol

```diff
@@ -0,0 +1,120 @@
+pragma solidity 0.8.9;
+
+import "forge-std/Test.sol";
+
+import { RevenueToken } from "../mock/RevenueToken.sol";
+import { LineFactory } from "../modules/factories/LineFactory.sol";
+import { ModuleFactory } from "../modules/factories/ModuleFactory.sol";
+import {SecuredLine} from "../modules/credit/SecuredLine.sol";
+import {Spigot} from "../modules/spigot/Spigot.sol";
+import {Escrow} from "../modules/escrow/Escrow.sol";
+import {LineLib} from "../utils/LineLib.sol";
+
+
+contract LineFactoryTest is Test {
+  SecuredLine line;
+  Spigot spigot;
+  Escrow escrow;
+  LineFactory lineFactory;
+  ModuleFactory moduleFactory;
+
+  address oracle;
+  address arbiter; 
+  address borrower; 
+  address swapTarget; 
+  uint ttl = 90 days;
+  address line_address;
+  address  spigot_address;
+  address escrow_address;
+
+
+
+  function setUp() public {
+
+    oracle = address(0xdebf);
+    arbiter = address(0xf1c0);
+    borrower = address(0xbA05);
+    swapTarget = address(0xb0b0);
+    
+
+    moduleFactory = new ModuleFactory();
+    lineFactory = new LineFactory(address(moduleFactory));
+    
+    line_address = lineFactory.deploySecuredLine(oracle, arbiter, borrower, ttl, payable(swapTarget));
+    line = SecuredLine(payable(line_address));
+
+    spigot_address = address(line.spigot());
+    spigot = Spigot(payable(spigot_address));
+
+    escrow_address = address(line.escrow());
+    escrow = Escrow(payable(escrow_address));
+
+  }
+
+  function test_deployed_lines_own_modules()  public {
+    assertEq(spigot.owner(), line_address);
+    assertEq(escrow.line(), line_address);
+  }
+  
+ function test_arbiter_cant_be_null() public {
+    address arbiter = line.arbiter();
+    assertTrue(arbiter != address(0x000));
+  }
+
+
+  function test_new_line_has_correct_spigot_and_escrow() public {
+     assertEq(spigot.owner(), line_address);
+     assertEq(escrow.line(), line_address);
+     assertEq(address(line.escrow()), address(escrow));
+     assertEq(address(line.spigot()), address(spigot));
+  }
+
+  function test_revenue_split_cannot_exceed_100() public {
+    assertEq(line.defaultRevenueSplit() < 100, true);
+  }
+
+  function test_status_active() public {
```

Comment:
can u change test name to "test_newly_deployed_lines_are_always_active"

Reply:
done